Gems/Archive/Code/Source/Clients/ArchiveTOCView.h

```diff
@@ -105,5 +105,48 @@ namespace Archive
         const ArchiveTableOfContentsView& tocView);
 } // namespace Archive
 
+// Separate namespace block to create visual spacing around utility functions
+// for querying the the Table of Contents View
+namespace Archive
+{
+   //! Retrieves a subspan from the Table of Contents Archive BlockLine Offset Table
+   //! that contains the block lines associated with a specific file
+   //! @param fileMetadataTableIndex index into the file metadata table
+   //!        This is used to lookup the uncompressed size and the first block line offset entry
+   //!        for a file
```

Comment:
@params here don't match the declaration

Reply:
The missing tocView parameter has been added.